source/features/use-first-commit-message-for-new-prs.tsx

```diff
@@ -6,17 +6,23 @@ import * as textFieldEdit from 'text-field-edit';
 import features from '.';
 import looseParseInt from '../helpers/loose-parse-int';
 
-function getFirstCommitMessage(): string[] {
-	const commitSummaryWrapper = select('.js-commits-list-item a.Link--primary')!.parentElement!;
-	const commitDescription = select('.js-commits-list-item pre')?.textContent ?? '';
+function formattedContent(node: ChildNode): string {
```

Comment:
Nit pick can you move this function under `reformattedTitle`, it took me a few seconds to figure out what this was doing here and why it did not have a comment.